receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/accumulator.go

```diff
@@ -43,15 +44,27 @@ func (acc *metricDataAccumulator) getMetricsData(containerStatsMap map[string]*C
 		stats, ok := containerStatsMap[containerMetadata.DockerID]
 
 		if ok && !isEmptyStats(stats) {
+
 			containerMetrics := convertContainerMetrics(stats, logger, containerMetadata)
 			acc.accumulate(convertToOTLPMetrics(ContainerPrefix, containerMetrics, containerResource, timestamp))
 			aggregateTaskMetrics(&taskMetrics, containerMetrics)
-			overrideWithTaskLevelLimit(&taskMetrics, metadata)
-			acc.accumulate(convertToOTLPMetrics(TaskPrefix, taskMetrics, taskResource, timestamp))
+
+		} else if containerMetadata.FinishedAt != "" && containerMetadata.StartedAt != "" {
+
+			duration, err := calculateDuration(containerMetadata.StartedAt, containerMetadata.FinishedAt)
+
+			if err != nil {
+				logger.Warn("Error time format error found for this container:" + containerMetadata.ContainerName)
+			}
+
+			acc.accumulate(convertStoppedContainerDataToOTMetrics(ContainerPrefix, containerResource, timestamp, duration))
+
 		} else {
-			acc.accumulate(convertResourceToRMS(containerResource))
+			logger.Warn("Missing stats data for this docker id :" + containerMetadata.DockerID)
```

Comment:
I think maybe I should delete this warning, because of the format shouldn't be wrong.

Reply:
I think we can remove this block for another reason. Considering the expected behivor, if the first two blocks are false, we don't need to do anything. Rather, we may look to print some warning inside the above `if blocks` if needed. Also, the warnning message does not reflect the real reason. So, my suggestion would be to remove this `else` block.

Reply:
Yeah, else deleted.